src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -83,6 +121,23 @@ private static void AcceptButton_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        /// <summary>
+        /// This method will be executed when the Decline button is pressed in the TermsOfUseView Window and creates the exit tour modal
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        internal void DeclineButton_Click(object sender, RoutedEventArgs e)
+        {
+            var exitGuide = new ExitGuide();
+            exitGuide.Height = 273;
+            exitGuide.Width = 480;
```

Comment:
Can we set these as constants and why do we need to do these here?

Reply:
Yes, this is because we usually read from json here. I've updated setting those values to const variables.

Reply:
@filipeotero Can we read the setting again or get it from some other class? Usually, I prefer a single source of truth

Reply:
Oh, I see. Now is getting from JSON file.